tutorials/math/Bessel.py

```diff
@@ -0,0 +1,181 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Show the different kinds of Bessel functions available in ROOT
+## To execute the macro type in:
+##
+## ~~~{.cpp}
+## root[0] .x Bessel.C
+## ~~~
+##
+## It will create one canvas with the representation
+## of the  cylindrical and spherical Bessel functions
+## regular and modified
+##
+## Based on Bessel.C by Magdalena Slawinska
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TCanvas, TF1, gSystem, gPad, TLegend, TPaveLabel, kBlack
+
+
+gSystem.Load("libMathMore")
```

Comment:
Is this really needed or autoloading can do the job?

Reply:
Doesn't work with out this line.